localization/replace/german/dlc/fp2/dlc_fp2_major_decisions_l_german.yml

```diff
@@ -1,4 +1,4 @@
 ﻿l_german:
- secure_mediterranean_decision.a:0 "Unlocks the #italic Abecean Conquest#! [casus_belli|E], allowing you to conquer any county along the shores of the Abecean Sea for the duration of the [house|E] modifier while you control [mediterranean_title.GetName]."
- secure_mediterranean_decision.involved_notif:0 "[mediterranean_conqueror.GetPrimaryTitle.GetBaseName] Secured The Abecean"
- secure_mediterranean_decision.involved_notif_desc:0 "The [mediterranean_house.GetName] have unified various islands such as [mallorca.GetNameNoTier], [sardinas.GetNameNoTier] and [sicily.GetNameNoTier]. Declaring ownership over the Abecean and creating a valid [casus_belli|E] against your coastal Abecean counties."
+ secure_mediterranean_decision.a:0 "Schaltet die #italic Eroberung des Abekäischen Meeres# [casus_belli|E] frei, die es Ihnen erlaubt, eine beliebige Grafschaft entlang der Küste des Abekäischen  Meeres für die Dauer des [house|E] Modifikators zu erobern, während Sie [mediterranean_title.GetName] kontrollieren."
+ secure_mediterranean_decision.involved_notif:0 "[mediterranean_conqueror.GetPrimaryTitle.GetBaseName] Das Abekäische Meer ist gesichert!"
+ secure_mediterranean_decision.involved_notif_desc:0 "Die [mediterranean_house.GetName] hat verschiedene Inseln wie [mallorca.GetNameNoTier], [sardinas.GetNameNoTier] und [sicily.GetNameNoTier] geeint. Sie erklären den Anspruch auf das Abekäische Meer und können einen gültigen [casus_belli|E] gegen die Gemeinden an der Abekäischen-Küste errichten."
```

Comment:
Die [mediterranean_house.GetName] haben verschiedene Inseln wie [mallorca.GetNameNoTier], [sardinas.GetNameNoTier] und [sicily.GetNameNoTier] geeint. Sie erklären ihren Anspruch über die Abekäis was einen gültigen [casus_belli|E] gegen eure Grafschaften an der Küste des Abekäischen Meers erzeugt.